packages/hwloc/hwloc.spec

```diff
@@ -0,0 +1,78 @@
+Name: %{_cross_os}hwloc
+Version: 2.12.2
+Release: 1%{?dist}
+Summary: Portable hardware locality library
+URL: http://www.open-mpi.org/projects/hwloc/
+License: BSD-2-Clause
+Source0: https://download.open-mpi.org/release/hwloc/v2.12/hwloc-%{version}.tar.bz2
+
+BuildRequires: %{_cross_os}glibc-devel
+BuildRequires: %{_cross_os}systemd-devel
+
+%description
+%{summary}.
+
+%package devel
+Summary: hwloc development libraries and headers
+Requires: %{name}
+
+%description devel
+%{summary}.
+
+%prep
+%autosetup -n hwloc-%{version} -p1
+
+%build
+%cross_configure \
+    --disable-cairo \
+    --disable-gl \
+    --disable-libudev \
+    --disable-libxml2 \
+    --disable-opencl \
+    --disable-pci \
+    --disable-plugins \
+    --enable-static \
+    --disable-shared \
+    --exec-prefix=%{_cross_prefix} \
+    --program-prefix=""
+
+%force_disable_rpath
+
+%make_build
+
+%install
+%make_install
+
+find %{buildroot} -name '*.la' -exec rm -f {} ';'
+find %{buildroot} -name '*.debug' -exec rm -f {} ';'
+
```

Comment:
The `.la` files should be removed by `brp-remove-la-files` automatically.

Why are there `*.debug` files that need to be removed?